cunit/slowio.testc

```diff
@@ -112,6 +112,25 @@ static void test_limit2(void)
     }
 }
 
+void busywait_for(struct timespec *howlong)
+{
+    struct timespec start;
+    long long int max_tries = LLONG_MAX;
```

Comment:
I prefer to use stdint.h types these days for general integers that are more complicated than just `int` or `unsigned` (you might need to `#include <stdint.h>` if it's not already):

```suggestion
    int64_t max_tries = INT64_MAX;
```

If the intention was not specifically "64-bit integer", but more like "the biggest integer the platform has", look at `intmax_t`.  It's usually 64 bits too, but the different name communicates that we don't care about the exact number of bits so long as it's very big.